corehq/apps/app_manager/forms.py

```diff
@@ -63,3 +64,76 @@ def clean(self):
         if self.cleaned_data.get('linked'):
             if not LINKED_APPS.enabled(domain):
                 raise forms.ValidationError("The target project space does not have linked apps enabled.")
+
+
+class PromptUpdateSettingsForm(forms.Form):
```

Comment:
I will update the colour to be consistent. Regarding making it update the state, this page will be rarely accessed, and there isn't lot of dynamic settings (unlike in form-builder) that would usually make it difficult for the user to keep track of whether they updated something or not. Given this is a much simpler form for now, I think this is okay. This is also under a feature flag